tests/eggcc-extraction.egg

```diff
@@ -1494,18 +1520,21 @@
 (function context-ival (Operand Body) Interval
   :merge (interval-intersect old new))
 
-(rule ((= lhs (Node (PureOp (Const (BoolT) (const) (Bool b))))))
-      ((set (ival lhs) (BoolI b b))))
+(rule ((= lhs (Node (PureOp (Const (BoolT) (const) (Bool b)))))
+       (= nval (BoolI b b)))
+      ((set (ival lhs) nval)))
 
-(rule ((= lhs (Node (PureOp (Const (IntT) (const) (Num n))))))
-      ((set (ival lhs) (IntI n n))))
+(rule ((= lhs (Node (PureOp (Const (IntT) (const) (Num n)))))
+       (= nval (IntI n n)))
+      ((set (ival lhs) nval)))
```

Comment:
`IntI` is a constructor, not a function
So this isn't a lookup and doesn't need to be changed